valohai_yaml/schema/deployment-node.yaml

```
@@ -20,6 +20,9 @@ properties:
     type: array
     items:
       "$ref": "./node-action.json"
+  on-error:
+    type: string
+    enum: ["stop-all", "continue"]
```

Seems `ErrorAction.STOP_NEXT` isn't represented here? (Maybe add a test for it?)
```suggestion
    enum: ["stop-all", "stop-next", "continue"]
```